src/components/section/NewEventCard.tsx

```diff
@@ -185,27 +186,48 @@ const JoinExistingEventSection = () => {
       >
         Join Event
       </ButtonWithState>
-    </>
+    </div>
   );
 };
 
-const eventOptions = ["Start New", "Join Existing"];
+const TabButton: React.FC<React.PropsWithChildren> = ({ children }) => {
```

Comment:
Why are we removing this practice out? I'd say it's better to separate text from code it self so we have a bit extra clarity and if we wanted to make changes we can make them more easily.

Reply:
I see it's prob cuz we are doing this elsewhere, uhmm leave it like this if u think it's alright

Reply:
I did this primarily because the component we are (Headless UI) using does not accept an array but a tree of elements, and the tab titles are actually related to contents, meaning that they might be edited together (at least we cannot add new tab titles without adding more tab pages), so I decided to put them inline.